src/content/1.2/code/scala/snippet11.scala

```diff
@@ -0,0 +1,3 @@
+sealed trait Bool
+final case object True extends Bool
+final case object False extends Bool
```

Comment:
Here the `final` modifier is redundant and may be removed on future versions of scala.
See [this](https://github.com/scala/bug/issues/11094) for reference.

BTW, Thanks for bring this amazing work to **Scala** 😄